painter/src/painter.rs

```diff
@@ -452,14 +452,42 @@ impl Painter {
     self
   }
 
-  pub fn draw_img(&mut self, img: ShareResource<PixelImage>) -> &mut Self {
-    self
-      .rect(&Rect::from_size(Size::new(
-        img.width() as f32,
-        img.height() as f32,
-      )))
-      .set_brush(img)
-      .fill();
+  /// Draw the image
```

Comment:
Does this API do drawing `src_rect` from image to `dst_rect` in the painter context?